src/Component/Facet/Range.php

```diff
@@ -48,6 +48,42 @@ public function getMinCount(): ?int
         return $this->getOption('mincount');
     }
 
+    /**
+     * @param string $tag
+     *
+     * @return \Solarium\Core\Configurable
+     */
+    public function setTag($tag)
```

Comment:
According to https://lucene.apache.org/solr/guide/6_6/faceting.html tags are generic facet parameters like exludes.
Therefore we should not implement them specifically for range facets. We should follow or improve or extend the pattern of The ExludeTagsInterface and ExcludeTagsTrait combo.

Reply:
doesn't that imply solarium needs a complete rework of how it deals with local params?

Reply:
https://github.com/solariumphp/solarium/issues/735

Reply:
can be modified once https://github.com/solariumphp/solarium/pull/736 is reviewed and merged